todo/services/task_service.py

```
@@ -174,10 +199,17 @@ def prepare_deferred_details_dto(cls, deferred_details_model: DeferredDetailsMod
-        user = UserRepository.get_by_id(user_id)
-        if user:
-            return UserDTO(id=str(user_id), name=user.name)
-        raise UserNotFoundException(user_id)
+        """
+        Prepare UserDTO from user_id, fetching real user data from repository.
+        """
+        try:
+            user_model = UserRepository.get_by_id(user_id)
+            if user_model:
+                return UserDTO(id=str(user_model.id), name=user_model.name)
+        except Exception:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)